test/datatypes.test.js

```diff
@@ -6,6 +6,8 @@
 'use strict';
 require('./init.js');
 var assert = require('assert');
+var semver = require('semver');
+var isWin = /^win/.test(process.platform);
```

Comment:
you should move this function out to a new support file -- test/helpers/platform.js and move the semver stuff out to another at test/helpers/mysqlVersion.js